frontends/common/parser_options.cpp

```diff
@@ -21,6 +21,7 @@ limitations under the License.
 
 #include <sys/stat.h>
 #include <sys/types.h>
+#include <sys/wait.h>
```

Comment:
what do we need from sys/wait.h?

Reply:
The include added to test/gtest/helpers.cpp is for the same reason too.